app/views/playlists.js

```diff
@@ -30,6 +29,11 @@ const Playlists = Mn.CompositeView.extend({
             'delete:playlist',
             this.deletedPlaylist
         );
+        this.listenTo(
+            application.appState,
+            'change:currentPlaylist',
+            this.playlistChanged
```

Comment:
As long as this one is a CopmositeView, you should probably pass the `application.appState` as its model, and add a modelEvent on change that trigger the render. Then, simply use the [`childViewOptions`](http://marionettejs.com/docs/v2.4.5/marionette.collectionview.html#collectionviews-childviewoptions) to pass to the childView if it's selected or not, so you can use it directly in the childView's [className](http://backbonejs.org/#View-extend) to set it.

The reason to it is that it avoid the `playlistChanged` callback that manipulate childs elements instead of itself, and prevent potential side effects as long as the childViews collection are re-rendered. It may laso be more performant coz' the re-render won't manipulate each item one-by-one but will use a document fragment to cache the rendering part. It'll also reuse old DOM node.